components/SlateRenderer/components/ContactCard.tsx

```diff
@@ -0,0 +1,29 @@
+import type { ContactNode } from '@prezly/slate-types';
+import React from 'react';
+
+import { ContactCard as ContactCardComponent } from '@/components';
+
+import styles from '../SlateRenderer.module.scss';
+
+interface ContactCardComponentProps {
```

Comment:
I suggest using `Props` naming in such cases. Especially when this interface is not exported publicly. I see the prefix as unnecessary visual noise.